hack/functest.sh

```diff
@@ -2,12 +2,24 @@
 
 set -e
 
-# Override the image name when this is invoked from openshift ci
+OPERATOR_NAME="cincinnati-operator"
+GRAPH_DATA_CONTAINER_NAME="cincinnati-graph-data-container"
+REGISTRY="quay.io"
+OPERATOR_IMAGE="${REGISTRY}/cincinnati/${OPERATOR_NAME}:latest"
+GRAPH_DATA_IMAGE="${REGISTRY}/ppinjark/${GRAPH_DATA_CONTAINER_NAME}:latest"
```

Comment:
Probably don't want to leave this pointing at your graph data image by default. Especially given that I don't believe it is overrideable in its current form.

Reply:
The value gets replaced to `quay.io/ppinjark/cincinnati-graph-data-container:latest` when running locally. Not sure what other value we need to use. Is there a generic/standard value we should use here?

Reply:
@djzager I have removed the dependency form the go config to create k8s resources. Now the resources are getting created from the deploy dir. Necessary replacements are made in the yaml files from the bash script hack/functest.sh

@sacharya @rwsu FYI